Readme.md

```diff
@@ -100,7 +100,7 @@ We recommend to use `cargo doc` to generate documents for each crate in this SDK
 
 # Sample Codes
 
-We provide five sample codes to help developers understand how to write Enclave codes in Rust. These codes are located at `samplecode` directory.
+We provide eighteen sample codes to help developers understand how to write Enclave codes in Rust. These codes are located at `samplecode` directory.
```

Comment:
For futureproofing, perhaps "many" would be better?

Reply:
LOL made my day! I'll update it to "many" in the next release!